src/libraries/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs

```diff
@@ -131,6 +132,7 @@ internal ECParameters ExportParameters(bool includePrivateParameters, int keySiz
         {
             // Apple requires all private keys to be exported encrypted, but since we're trying to export
             // as parsed structures we will need to decrypt it for the user.
+            // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Unit test password.")]
```

Comment:
This code, and a few other places, are not part of a unit test. Perhaps the comment above can be incorporated as the justification?

Reply:
Unfortunately it is not possible to use multiline justification. I added short justification with reference to the source.